src/main/java/org/spongepowered/api/entity/living/animal/HorseColor.java

```diff
@@ -42,4 +42,10 @@
      */
     String getName();
 
+    /**
+     * Checks if this is a flowerpot.
+     *
+     * @return Whether this is a flowerpot
+     */
+    boolean isFlowerPot();
```

Comment:
I can now finally easily tell if my Horse is colored FlowerPot, excellent feature! lol